autosubmit/job/job.py

```diff
@@ -1621,8 +1601,7 @@ def update_current_parameters(self, as_conf: AutosubmitConfig, parameters: dict)
 
     def update_platform_parameters(self, as_conf, parameters):
         if not self.platform:
-            submitter = job_utils._get_submitter(as_conf)
-            submitter.load_platforms(as_conf)
+            submitter = ParamikoSubmitter(as_conf=as_conf)
```

Comment:
Now we simply declare `ParamikoSubmitter` and pass the `as_conf` object -- when the object is passed, the constructor of the class will call `self.load_platforms`, simplifying a bit.